### PR TITLE
Questions page

### PR DESCRIPTION
**Pull request by AliciadW**

### CODE REVIEW

File: src/views/QuestionPage.jsx [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 const Markup = (props) => (
   <Fragment>
-    <h1>Questions Page</h1>
-    <Preview>
-      {JSON.stringify(props, 0, 2)}
-    </Preview>
+
+    <Helmet>
+      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"/>
+      <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```
Comment (schalkventer) [inline src/views/QuestionPage.jsx]: Is there a specific reason why you are using the Material Icons Font and not the SVG one through `import`?

File: src/views/StoryTemplate.js [hunk 2]
```diff
@@ -139,43 +48,42 @@ const QuestionActions = styled.div`
   margin-bottom: 40px;
 `;
 
-const StoryTemplate = () => (
-  <Fragment>
-    <Helmet>
-      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"/>
-      <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
-    </Helmet>
+class StoryTemplate extends Component {
```
Comment (schalkventer) [inline src/views/StoryTemplate.js]: Is there a specific reason why you converted this into a stateful component?

Comment (AliciadW) [reply src/views/StoryTemplate.js]: This was to be able to change the state when an input is given. I can change it back if this is not correct.

File: src/views/QuestionPage.jsx (hunk 1 above)
Comment (AliciadW) [reply src/views/QuestionPage.jsx]: No, would you prefer that I rather use the SVG one?

**Comment by AliciadW** [comment]
@schalkventer I'll merge this PR and address any concerns in the next PR.

**Pull request merged**